### DISCUSSION TITLE
New chain onboarding process?

### DISCUSSION THREAD

**Discussion by porkchop** [WalletConnect/walletconnect-monorepo#2495] [Developer Support] [answered]
Hi I am just wondering about the steps to introduce a new chain. I see that we are supposed to submit an issue to have our chain included in the list (as per https://docs.walletconnect.com/2.0/advanced/multichain/overview). Below the registration instructions it says to add RPC support to the example wallet and dApp. Is that all there is too it? Am I missing anything? Thanks for any help. -Aaron

**Comment by finessevanes** [comment, marked as answer]
That's all there is to it! Ensure you complete the entire GitHub Issue.

> Below the registration instructions it says to add RPC support to the example wallet and dApp. Is that all there is too it?

Yes, that's correct. You can open a PR to both repositories, the dapp and wallet, to demonstrate the RPC methods in use.

**Reply by porkchop** [reply]
Thank you!
